### DISCUSSION TITLE
math: multiple in-line math (`$..$`) in one line doesn't render properly sometimes

### DISCUSSION THREAD

**Discussion by ShamrockLee** [community/community#18988] [Other Feature Feedback, Questions, & Ideas] [answered]
### Description
The rendering problem occurs to the input boxes, and it seems to occur randomly.

### Reproduction:

Raw:
```Markdown
The first implementation is $\mathcal O(n^2)$, while the second one $\mathcal O(n)$.
```

Result:
The first implementation is $\mathcal O(n^2)$, while the second one $\mathcal O(n)$.

**Comment by charleprr** [comment]
You don't need the `\mathcal` command.

Raw:
`The first implementation is $O(n^2)$, while the second one $O(n)$.`

Result:
The first implementation is $O(n^2)$, while the second one $O(n)$.

**Reply by ShamrockLee** [reply]
Oops! Probably not here.

But how about this one:

Raw:
```Markdown
The natural number set $\mathbb N$ is a subset of the integer set $\mathbb Z$
```

Result:
The natural number set $\mathbb N$ is a subset of the integer set $\mathbb Z$

**Reply by charleprr** [reply]
Oh, indeed! Looks like a bug...

**Comment by mdickinson** [comment]
I'm seeing the same with:

```
let $\alpha$ and $\beta$ be Greek letters
```

Result:

let $\alpha$ and $\beta$ be Greek letters

And screenshot of the result, for posterity:

<img width="211" alt="Screenshot 2022-06-22 at 08 35 48" src="https://user-images.githubusercontent.com/662003/174971266-6011a2a5-e386-47d2-a1c4-5ed600a7c8c4.png">

**Comment by nschloe** [comment, marked as answer]
This has been fixed now.
